### DISCUSSION TITLE
Training Log 2023-03-18

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#395] [Training Logs 训练日志]
# CPM-Live Training Log (March, 18)

Time: March, 18 2023 19:00

Recorder: @zh-zheng

## Loss
- Begin: 1.73
- End: 1.75
	
## Completed Data
- $\approx$ 2886.72GB

## Average Grad Norm
- 2.70

## Progress
- 98.56%

## Comment

Today, we test the performance of our model with a what if question.

**Comment by allen0125** [comment]
@zh-zheng 想请教下这里的训练日志是自动生成的还是手动填写的呢？如果是自动生成的话，能分享下实现方法吗？感谢！
